src/extension.ts

```diff
@@ -543,8 +543,6 @@ export class ExtensionManager implements vscode.Disposable {
                 }
             }
         }
-
-        this.updateCodeModel(project);
```

Comment:
Can you double check that the `ConfigureTrigger.configureWithCache` case continues to work if you remove this? I walked through the code and I think it will still fire the `onCodeModelChanged` event, but I didn't test it explicitly.

The scenario is `cmake.configureOnOpen` set to false and the user has manually configured at least once with CMake Tools.  In this scenario the file-api data is available and we can update the code model.  That should trigger the event that then calls `this.updateCodeModel`.

Reply:
I do believe I've tested this case and confirmed it works, but I'll make a note to test this again explicitly to ensure it does.

Reply:
Just tested again and confirmed this works.